app/views/layouts/_footer.html.erb

```diff
@@ -2,12 +2,14 @@
   <div class="container-fluid">
     <div class="clearfix">
       <ul class="ui-ul list-inline pull-left">
-        <li>&copy; 2015 JOLT Labs LLC and individual contributors
-        <li> &nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;</li>
-        <li><%= mail_to 'info@shescoding.org','info@shescoding.org' %></li>
+        <li>
+          &copy; 2015 - <%= Time.current.year %> JOLT Labs LLC and individual contributors
```

Comment:
Yep, I added time.current.year so it's always up to date :D

Reply:
Perfect.